packages/sdk/src/hooks/useEnvWeb3.ts

```diff
@@ -12,7 +12,7 @@ export interface RPC {
 
 export const defaultRPC = {
   [SupportedChainId.MAINNET]: sample(["https://cloudflare-eth.com", "https://rpc.ankr.com/eth"]),
-  [SupportedChainId.FUSE]: sample(["https://rpc.fuse.io", "https://fuse-pokt.nodies.app/"])
```

Comment:
i dont think we need this change anymore, you can add liquify to the list though